packages/react/spec/auto/hooks/useTable.spec.tsx

```diff
@@ -527,6 +529,39 @@ describe("useTable hook", () => {
   });
 
   describe("custom cell renderer", () => {
+    // This test makes sure that the custom cell renderer is a valid React component by trying to render it.
+    // If it is not a valid React component, it will throw an error.
+    it("should be able to pass a JSX element with a hook inside", () => {
```

Comment:
Without the fix introduced in the `helper.tsx` file, this test throws a lot of React-related errors.